CONTRIBUTING.md

```diff
@@ -49,17 +49,18 @@ working on your contributions.
 
 ### Format of the Commit Message
 
-We follow a rough convention for commit messages that is designed to answer two
-questions: what changed and why. The subject line should feature the what and
-the body of the commit should describe the why.
+The commit messages should be designed to answer two questions: what changed and why. 
+
+The subject line should consist of the modified subsytem and an imperative sentence
```

Comment:
Want to draft a proposed set? :-)

On Tue, Apr 19, 2016 at 2:02 PM, Tamer TAS <notifications@github.com> wrote:

> In CONTRIBUTING.md
> <https://github.com/coreos/rkt/pull/2461#discussion_r60217872>:
>
> > @@ -49,17 +49,18 @@ working on your contributions.
> >
> >  ### Format of the Commit Message
> >
> > -We follow a rough convention for commit messages that is designed to answer two
> > -questions: what changed and why. The subject line should feature the what and
> > -the body of the commit should describe the why.
> > +The commit messages should be designed to answer two questions: what changed and why.
> > +
> > +The subject line should consist of the modified subsytem and an imperative sentence
>
> @jonboulle <https://github.com/jonboulle> I think 1:1 is a great idea. it
> also saves developers from thinking about what to put there.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/2461/files/fc5783b2bb9abb4aea1cbf85ca715e9597d72d7a#r60217872>
>